### PR TITLE
Log config

### PR DESCRIPTION
**Pull request by chrisahl**
For issue https://issues.redhat.com/browse/AAP-10058, which is for AAP 2.3 on GCP with containerized installer, but it may also apply to current AAP 2.2 on GCP.  Still investigating

### CODE REVIEW

**Comment by chrisahl** [comment]
I deployed our current GCP marketplace offering and these changes also apply to that.  So we should be able to merge these changes with other 2.2.1 changes

**Review by ianf77 — COMMENTED** [review]
Just  one change

File: stories/assembly-aap-gcp-monitoring-logging.adoc [hunk 1]
```diff
@@ -5,14 +5,14 @@ ifdef::context[:parent-context: {context}]
 
 :context: aap-gcp-monitor-log
 
-You can send metrics to the {GCP} monitoring system to be visualized in the {GCP} UI. 
-{PlatformNameShort} from {GCPMarket} metrics and logging are disabled by default as there is a cost to send these metrics to GCP. 
+You can send metrics to the {GCP} monitoring system to be visualized in the {GCP} UI.
+{PlatformNameShort} from {GCPMarket} metrics and logging are disabled by default as there is a cost to send these metrics to GCP.
```
Comment (ianf77) [inline stories/assembly-aap-gcp-monitoring-logging.adoc:9]: {AAPonGCP} metrics

File: stories/assembly-aap-gcp-monitoring-logging.adoc [hunk 2]
```diff
@@ -5,14 +5,14 @@ ifdef::context[:parent-context: {context}]
 
 :context: aap-gcp-monitor-log
 
-You can send metrics to the {GCP} monitoring system to be visualized in the {GCP} UI. 
-{PlatformNameShort} from {GCPMarket} metrics and logging are disabled by default as there is a cost to send these metrics to GCP. 
+You can send metrics to the {GCP} monitoring system to be visualized in the {GCP} UI.
+{PlatformNameShort} from {AAPonGCP} metrics and logging are disabled by default as there is a cost to send these metrics to GCP.
```
Comment (ianf77) [inline stories/assembly-aap-gcp-monitoring-logging.adoc:9]: Apologies. I ought to have been clearer. You only need {AAPonGCP}, it replaces all of {PlatformNameShort} on {GCPMarket} and is much quicker to type

Comment (chrisahl) [reply stories/assembly-aap-gcp-monitoring-logging.adoc]: Try 2!

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**